[1793] Adding Solution ofmaximum-score-of-a-good-subarray.java

12345rams opened the pull request:
## Issue Description
#164

### Summary
This pull request addresses the issue titled "Maximum Score of a Good Subarray " Given an array of integers nums (0-indexed) and an integer k.

The score of a subarray (i, j) is defined as min(nums[i], nums[i+1], ..., nums[j]) * (j - i + 1). A good subarray is a subarray where i <= k <= j.

Return the maximum possible score of a good subarray.

## Proposed Changes
- Added a Java solution to find Maximum Score of a Good Subarray

12345rams committed:
Adding Solution ofmaximum-score-of-a-good-subarray.java

godkingjay reviewed (changes requested):
Please follow our directory structure:

```
LeetCode/
├── [DIFFICULTY]/
│   ├── [PROBLEM]/
│   │   ├── README
│   │   ├── solution.java
│   │   ├── solution.cpp
```

Thank you.

12345rams committed:
Delete Hard/maximum-score-of-a-good-subarray.java

12345rams committed:
Solution Of Maximum_score_of_a_good_subarray

In Format  
LeetCode/
├── [DIFFICULTY]/
│   ├── [PROBLEM]/
│   │   ├── README
│   │   ├── solution.java
│   │   ├── solution.cpp

12345rams commented:
I corrected all these mistake

godkingjay committed:
Update and rename README.md to README.md

godkingjay committed:
Update and rename solution.java to Solution.java
